### ISSUE TITLE
Dataloader function is missing

### ISSUE THREAD

**Issue by SergioRodLla** [BioMedIA-MBZUAI/HECKTOR2025#2]
Hi!

Regarding Task 1, am following your example code at "train.py" and I cannot find the "get_dataloaders()" function. It seems that the "/data" directory is hidden at the gitignore file. Could you provide it? 🙏 

Best,
Sergio

**Comment by shahdhardn** [comment]
Hello, done, please check

**Comment by SergioRodLla** [comment]
Hi, 

First of all, thank you for adding the files.
I saw that that you are using preprocessed .npz files when loading the dataset. Could you explain/show how you do this preprocessing? That would be very helpful.

Thank you again,

**Comment by AhmedAtefAhmedAly** [comment]
Hello Sergio, 

You can find find an example of preprocessing under the docker-template branch for task 1 [here](https://github.com/BioMedIA-MBZUAI/HECKTOR2025/blob/docker-template/Task_1/resources/preprocess.py). This code preprocess a single case for inference. For training purposes we processed the training dataset and saved them as npz files. 

Feel free to experiment with any preprocessing techniques but keep in mind that the resolution of the output mask should be the same as the original CT resolution.

**Comment by SergioRodLla** [comment]
Understood, thank you.
